app/Http/Controllers/Controller.php

```diff
@@ -16,6 +16,7 @@ public function index()
         $arr = [
             'key1' => 1,
             'key2' => 2,
+            'key3' => 3
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Controller.php</br>**Line**: 19</br>Each line in an array declaration must end in a comma